Makefile_packaging.mk

```diff
@@ -210,22 +211,87 @@ debs: $(DEBS)
-	sudo build $(BUILD_OPTIONS) $(ADD_REPOS) --repo zypp:// \
-	  --dist $(DISTRO_ID) $(RPM_BUILD_OPTIONS)
+sle12_REPOS += --repo https://download.opensuse.org/repositories/science:/HPC/openSUSE_Leap_42.3/     \
+	       --repo http://cobbler/cobbler/repo_mirror/sdkupdate-sles12.3-x86_64/                 \
+	       --repo http://cobbler/cobbler/repo_mirror/sdk-sles12.3-x86_64                        \
+	       --repo http://download.opensuse.org/repositories/openSUSE:/Backports:/SLE-12/standard/ \
+	       --repo http://cobbler/cobbler/repo_mirror/updates-sles12.3-x86_64                    \
+	       --repo http://mgmt-1.wolf.hpdd.intel.com/cobbler/pub/SLES-12.3-x86_64/
+
+sl42_REPOS += --repo https://download.opensuse.org/repositories/science:/HPC/openSUSE_Leap_42.3 \
+	      --repo http://download.opensuse.org/update/leap/42.3/oss/                         \
+	      --repo http://download.opensuse.org/distribution/leap/42.3/repo/oss/suse/
+
+sl15_REPOS += --repo http://download.opensuse.org/update/leap/15.1/oss/            \
+	      --repo http://download.opensuse.org/distribution/leap/15.1/repo/oss/
+
+chrootbuild: $(SRPM) Makefile
+	add_repos="";                                                       \
+	for repo in $(ADD_REPOS); do                                        \
+	    if [[ $$repo = *@* ]]; then                                     \
+	        branch="$${repo#*@}";                                       \
+	        repo="$${repo%@*}";                                         \
+	    else                                                            \
+	        branch="master";                                            \
+	    fi;                                                             \
+	    case $(DISTRO_ID) in                                            \
+	        sle12.3) distro="sles12.3";                                 \
+	        ;;                                                          \
+	        sl42.3) distro="leap42.3";                                  \
+	        ;;                                                          \
+	        sl15.1) distro="leap15.1";                                  \
+	        ;;                                                          \
+	    esac;                                                           \
+	    baseurl=$${JENKINS_URL}job/daos-stack/job/$$repo/job/$$branch/; \
+	    baseurl+=lastSuccessfulBuild/artifact/artifacts/$$distro/;      \
+            add_repos+=" --repo $$baseurl";                                 \
+        done;                                                               \
+	sudo build --nosignature $(BUILD_OPTIONS) $$add_repos               \
+	     $($(basename $(DISTRO_ID))_REPOS) --dist $(DISTRO_ID) $(RPM_BUILD_OPTIONS) \
+	     $(SRPM)
 endif
 
+docker_chrootbuild:
+	docker build --build-arg UID=$$(id -u) -t $(BUILD_OS)-chrootbuild \
+	             -f packaging/Dockerfile.$(BUILD_OS) .
+	docker run --privileged=true -w /home/brian/daos/rpm/openpa           \
```

Comment:
Ahhh.  Hard-coded path.  Should be `$$PWD`.  I'll fix that.  But the `docker_chrootbuild` target is meant to stay, so that one can do local docker chroot build tests.